#Lissajous
Amp1 : 0.2
freq1 : 0.5
Amp2 : 0.2
freq2 : 1
phi : 0
x0 : 0.5 
y0 : 0
z0 : 0.4